From 86680e93beaef5d1c55c928c62e94a903a3c2e58 Mon Sep 17 00:00:00 2001
From: zengsj <zengshuangjie@gmail.com>
Date: Tue, 11 Jun 2019 14:16:29 +0800
Subject: [PATCH] fix-build-with-openssl

---
 usrsctp/BUILD.gn | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/usrsctp/BUILD.gn b/usrsctp/BUILD.gn
index 94f26786e1d..1cb95a02bd9 100644
--- a/usrsctp/BUILD.gn
+++ b/usrsctp/BUILD.gn
@@ -141,6 +141,7 @@ static_library("usrsctp") {
     defines += [ "__Userspace_os_Fuchsia" ]
   }
   deps = [
-    "//third_party/boringssl",
+    # "//third_party/boringssl",
+    "//yealink/third_party/openssl:openssl",
   ]
 }
-- 
2.15.1.windows.2

